Coalition: Long Lost Property (#6340)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			choice
				`	"Alright, I'll go look for it."`
					goto accept
				`	"What if I can't find it in the system? If it hasn't arrived, would I need to wait a long time for it to get there?"`
					goto wait
				`	"What's in it for me?"`
			`	"Pay you handsomely for the retrieval, House Idriss would," she answers. "Also, if of any use, the data is, use it in some form of long-ranged scanners, we could. Willing to sell such scanners to you, we would be."`
				goto accept
			label wait
			`	"Not too long," the short Arach answers. "Minor changes in its speed, the probe might have had, but delay its arrival for more than a few weeks, it could not. One week from now, the absolute limit date for its arrival is."`
			`	"If locate it immediately, you do not, maybe ask the local humans you could," the taller one adds. "Newsworthy in most systems, such an event would be."`
			label accept

> 			`	They show you an image of the probe, which looks pretty much like an ordinary satellite, and say they'll work on your ship's scanning system to pick up the object's signature easier. Flopug says goodbye, telling you she must continue working, but has the two Arachi bring you back to your ship and adjust your scanners. They wish you good luck once they're done, and leave your ship.`

Reviewer: ```suggestion
			`	They show you an image of the probe, which looks pretty much like an ordinary satellite, and say they'll work on your ship's scanning system to pick up the object's signature easier. Flopug says goodbye and tells you she must continue working, but has the two Arachi bring you back to your ship and adjust your scanners. They wish you good luck once they're done, and leave your ship.`
```

			choice
				`	"When I'm done they won't have a fleet to come here. A hundred thousand should be enough of a tip for pointing me their way."`
				`	"Fine, just tell me already."`
					goto bribe
			branch "convince" "fail"
				random < 20
			label convince
			apply
				payment -100000
			`	He stares at you for a few seconds, and lets out a laugh. "Well, it was worth a try. Alright, a hundred thousand it is."`
				goto where
			label fail
			`	"Now I can't just trust you to get every last one of their ships, can I?" He replies. "If you want to make it worth for me, it's a million. If you're not interested, there's nothing I can do about it."`
			choice

> 				`	"Fine, just tell me already."`

Author: That's sort of the same issue in my third bulletpoint, if there was an option to defer you'd get the conversation repeating when you got it again, unless I had a condition checking if you had gotten to this point previously, but I don't know if I should do that as it seems like a poor solution.

			label bribe
			apply
				payment -1000000
			`	"Thank you for your patronage," he says, and sits back down.`
			label where
			`	"The men who approached me for the machine are pirates, I imagine you had that figured out," he begins. "They aren't from the south though, I could tell you that much. Older, better dressed. Said they had read the reports about the 'unknown probe' entering Han, and wanted it for their business. I asked how they intended to use it, nobody here had figured how it worked, but they said they'd find a way to make use of it, have it warn them of incoming fleets in advance."`
			choice
				`	"What do you think they meant by that?"`
					goto meant
				`	"Did you catch their names?"`

> 			`	"Did they tell me their names? Yes. Their real ones? Doubt it, they seemed organized enough they wanted to keep that information hidden. I did get a look at their ships while we were loading the product into one of them, though."`

Reviewer: ```suggestion
			`	"Did they tell me their names? Yes. Their real ones? Doubt it, they seemed organized enough that they wanted to keep that information hidden. I did get a look at their ships while we were loading the product into one of them, though."`
```

				launch
	to complete
		never
	to fail
		has "Coalition: Long Lost Property 3: done"

mission "Coalition: Long Lost Property 4"
	name "Comparing Binaries"
	description "Head to <waypoints> to gather data on the binary stars, so that House Idriss may compare it with their probe's own predictions."
	to offer
		has "Coalition: Long Lost Property 3: done"
	source "Ablub's Invention"
	waypoint "Alnasl"
	on offer
		conversation
			`The Arach duo greet you in the spaceport and ask you to bring them inside your ship to talk. They look to the sides and behind them every other moment, as if watching out for something. When you're inside your ship's command room, the taller one speaks. "Compared the probe's prediction with its last reading of the system of arrival, we have. A very good approximation to real conditions, it reached. Though incredibly outdated, its hardware is, an ingenious program, what my ancestors-"`
			`	"My ancestors," the short one interrupts him.`
			`	"What our ancestors created is."`

> 			`	They show you a device with a map on screen, depicting nearby star systems, and the short Arach starts explaining, "Intrigued as to just how accurate its predictions have become, we are, so to verify its data about a star deeper in human space, the next step is." He points to a system at the edge of the map. "Have humans discovered this system, Captain? Interested in comparing the probe's data on far away binary stars, the president is."`

Reviewer: ```suggestion
			`	They show you a device with a map on screen, depicting nearby star systems, and the short Arach starts explaining. "Intrigued as to just how accurate its predictions have become, we are, so to verify its data about a star deeper in human space, the next step is." He points to a system at the edge of the map. "Have humans discovered this system, Captain? Interested in comparing the probe's data on far away binary stars, the president is."`
```

> 			`	You look at the screen and at your own map for a moment, and recognize the system as <waypoints>. "Settled it is, then. Head to that system, you should," the taller one speaks again. "Still with our adjustments, your scanners are, so just tinker them a bit to gather the needed, we will."`

Reviewer: ```suggestion
			`	You look at the screen and at your own map for a moment, and recognize the system as <waypoints>. "Settled it is, then. Head to that system, you should," the taller one speaks again. "Still with our adjustments, your scanners are, so just tinker them a bit to gather the needed data, we will."`
```

> 			`	A few minutes later, they're done with your scanners, and leave your ship, wishing you good luck.`

Reviewer: ```suggestion
			`	Once they're done with your scanners a few minutes later, they leave your ship and wish you good luck.`
```

				accept
	on enter "Alnasl"

> 		dialog `One of your monitors brings up various sets of data written in the Arach language when you enter the system, and your scanners begin gathering the data House Idriss wants. A few minutes pass, and just when it seems to end, a loading bar comes up. When it reaches the end, another one appears, and the process repeats a few more times. After about an hour, your monitor goes back to normal.`

Reviewer: ```suggestion
		dialog `One of your monitors brings up various sets of data written in the Arach language when you enter the system, and your scanners begin gathering the data House Idriss wants. A few minutes pass untill a loading bar comes up. When it reaches the end, another one appears, and the process repeats a few more times. After about an hour, your monitor goes back to normal.`
```

	on complete
		event "outskirts gauger available" 25 63
		payment 250000
		conversation
			`Flopug is waiting by the landing bays for you, with the Arach duo beside her. They come inside your ship when you've landed, and start checking the data, after Flopug pays you <payment>.`
			`	"Umm... more errors with this one, the probe committed," the taller Arach says. "Gotten a good deal right, it still did, but a shame it is, that deal with that much more of a distance, it couldn't."`
			`	"Not distance, the cause of errors was," the short Arach responds. "Confused with the binaries' signatures, it got. Refined to account for binary star systems and their oddities, its algorithms had to be."`
			`	"At any rate, happy to call the probe's voyage a success, I am," Flopug cuts them off. "Expected it to have developed this well, we hadn't, when notified of its existence by... I mean, when remembered about the approaching date of its arrival to that system, we did."`

> 			`	They turn you scanners back to normal, and tell you they'll work on using the data to build a new scanner, and will let you know about it when it's ready, the next time you're on <origin>. They thank you for your help again, and leave your ship.`

Reviewer: ```suggestion
			`	They turn you scanners back to normal, telling you that they'll work on using the data to build a new scanner.  They also tell you that they will let you know about it when it's ready, the next time you're on <origin>. They thank you for your help again, and leave your ship.`
```

event "outskirts gauger available"
	planet "Ablub's Invention"
		outfitter "Outskirts Gauger"

mission "Coalition: Outskirts Gauger Ready"
	landing
	invisible

> 	to offer

Reviewer: Perhaps this should have some sort of timer? As it stands it's possible for this mission to trigger the very next day, which is probably not in line with designing and building a new scanner.

Author: There's `event "outskirts gauger available" 25 63` on line 2094, is that not working properly? I thought that's how you did it to have a range for the timer, this one being from 25 to 63 days.

Reviewer: Oops, I completely missed that. Feel free to resolve.

		has "event: outskirts gauger available"
	source "Ablub's Invention"
	on offer
		conversation

> 			`When you leave your ship, you're greeted by Flopug, of House Idriss. "Completed work on our newest set of scanners, we have. Since helped us with recovering the probe and its data, you have, only fair, it is, that allowed access to the equipment, you are."`

Reviewer: ```suggestion
			`When you leave your ship, you're greeted by Flopug of House Idriss. "Completed work on our newest set of scanners, we have. Since helped us with recovering the probe and its data, you have, only fair, it is, that allowed access to the equipment, you are."`
```

			choice
				`	"When I'm done they won't have a fleet to come here. A hundred thousand should be enough of a tip for pointing me their way."`
				`	"Fine, just tell me already."`
					goto bribe
			branch "convince" "fail"
				random < 20
			label convince
			apply
				payment -100000
			`	He stares at you for a few seconds, and lets out a laugh. "Well, it was worth a try. Alright, a hundred thousand it is."`
				goto where
			label fail
			`	"Now I can't just trust you to get every last one of their ships, can I?" He replies. "If you want to make it worth for me, it's a million. If you're not interested, there's nothing I can do about it."`

> 			choice

Author: Added the comment.
